docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+
+32 kbytes & 4 ways is not feasible with the current architecture. Other size/ways configurations may be implemented in the design.
+
+The design will support one replacement policy allowed by L1I-50.
+
+=== FENCE.T custom instruction
+
+There are discussions within RISC-V International to define a specification for `FENCE.T`. The specification below reflects the situation prior to this RISC-V specification, based on Nils Wistoff’s work. If a RISC-V specification is ratified, the CVA6 specification will likely switch to it.
+
+|===
+
+|FET&#8209;10|CVA6 shall support the `FENCE.T` instruction that ensures that the execution time of subsequent instructions is unrelated with predecessor instructions.
+|FET&#8209;20|`FENCE.T` shall be available in all privilege modes (machine, supervisor, user and hypervisor if present).
+|===
+FENCE.T goes beyond `FENCE` and `FENCE.I` as it clears L1 caches, TLB, branch predictors... It is a countermeasure for SPECTRE-like attacks. It is also useful in safety-critical applications to increase execution time predictability.
+
+It is not yet decided if the `FENCE.T` instruction arguments can be used to select a subset of microarchitecture features that will be cleared. The list of arguments, if any, will be detailed in the user’s guide.
+
+Anticipation of verification: It can be cumbersome to prove the timing decorrelation as expressed in the requirement with digital simulations. We can simulate the microarchitecture features and explain how they satisfy the requirement as Nils Wistoff’s work demonstrated.
+
+== PPA targets
+
+These PPA targets will likely be updated when performance monitoring is integrated in the continuous integration flow.
+
+|===
+
+|PPA&#8209;10|CVA6 should be resource-optimized on FPGA and ASIC targets.
+|PPA&#8209;20|CVA6 should deliver more than 2.1 CoreMark/MHz.
+|PPA&#8209;30|CV32A6 should run at more than 150 MHz in the cv32a6_imac_sv32 configuration on Kintex 7 FPGA technology, commercial -2 speed grade.
+|PPA&#8209;40|CV64A6 should run at more than 900 MHz in the cv64a6_imacfd_sv39 configuration on 28FDSOI technology in the worst case frequency corner with the fastest threshold voltage.
+|PPA&#8209;50|TBD: Placeholder for single-precision floating performance per MHz.
+|PPA&#8209;60|TBD: Placeholder for double-precision floating performance per MHz.
+|===
+
+== Interface requirements
+
+=== Memory bus
+
+|===
+
+|MEM&#8209;10|CVA6 memory interface shall comply with AXI5 specification including the Atomic_Transactions property support as defined in [AXI] section E1.1.
+|MEM&#8209;20|CVA6 AXI memory interface shall feature user bit extensions on the data bus (`WUSER` and `RUSER` as per [AXI]) in connection with the L1I and L1WTD datasize extensions, with a number of user bits greater or equal to 0.
+|===
+
+The interface usually complies with AXI4. However, Atomic_Transactions is only defined in AXI5. For the sake of clarity, we do not use the AXI5-Lite interface.
```

Comment:
what does "usually complies with AXI4" mean? is that just an informative comment that AXI5 is similar to AXI4 with the addition of Atomic_Transactions?

Reply:
Removed "usually"